src/nvim/fileio.c

```diff
@@ -289,7 +289,7 @@ readfile (
   int wasempty;                         /* buffer was empty before reading */
   colnr_T len;
   long size = 0;
-  char_u      *p = NULL;
+  uint8_t      *p = NULL;
```

Comment:
the whitespace was for alignment. Nothing is aligned here, so there is no need for more than 1 space now.

Reply:
fixed